Makefile

```diff
@@ -3,6 +3,8 @@ BUILDDIR = ./build
 
 all: vet fmt lint test build
 
+ci: clean vet lint test
```

Comment:
Seeing an error on the `lint` command - https://github.com/Automattic/go-search-replace/actions/runs/11247950493/job/31272247126?pr=41#step:4:8

Are errors getting ignored?

Reply:
Oh, golint is deprecated :)